packages/mint-components/src/components/tax-and-cash/sqm-user-info-form/sqm-user-info-form.feature

```diff
@@ -0,0 +1,60 @@
+@owner:andy
+@author:andy
+
+Feature: Tax Form Step One
+
```

Comment:
@Locrian24 @AndiLuo this spec is missing the new cases for what happens if the users email (used on the microsite) actually connects to a partner already in the system. There are some designs and we can talk more about them but lets get the spec process started